### PR TITLE
Add Default Category Value when Not Provided in Timeline Data

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Description
When timeline data is returned via the `return_outputs` convenience function, if the timeline dict(s) do not include the `"Category"` key and value, the dict(s) will be updated with that key set with the value of `"Integration Update"`.

## Minimum version of Demisto
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No - server version 5.5.0 hasn't been released yet.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by avidan-H** [comment]
Refreshed version of PR [here](https://github.com/demisto/content/pull/5990).

File: Packs/Base/Scripts/CommonServerPython/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ## [Unreleased]
-
-
-## [20.3.3] - 2020-03-18
```
Comment (Itay4) [inline Packs/Base/Scripts/CommonServerPython/CHANGELOG.md]: need to fix changelog

**Comment by avidan-H** [comment]
@kirbles19 can you take a look at the [CHANGELOG](https://github.com/demisto/content/edit/timeline-category-constant1/Packs/Base/Scripts/CommonServerPython/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F6242) please.

**Comment by kirbles19** [comment]
@avidan-H Done.

**Pull request merged**